megamek/resources/log4j2-release.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration name="release-configuration" status="off">
+
+    <!-- This is the release log4j2 configuration, intended for when end users run the app. -->
+
+    <properties>
+        <property name="logfile.filename">${sys:megamek.log.file:-./logs/mm.log}</property>
+        <property name="logfile.pattern">%8relative [%-30.30thread] %-5level %-40.40logger - %message{nolookups}%n%rThrowable</property>
+    </properties>
+
+    <appenders>
+        <file name="file" fileName="${logfile.filename}" append="false" createondemand="true">
+            <patternlayout pattern="${logfile.pattern}" disableAnsi="true" />
+        </file>
+    </appenders>
+
+    <loggers>
+        <logger name="megamek" level="info" />
+        <logger name="System"  level="info" />
+        <root level="warning">
```

Comment:
Console message when running from a Jar:
2018-09-08 10:31:06,947 main WARN Error while converting string [warning] to type [class org.apache.logging.log4j.Level]. Using default value [null]. java.lang.IllegalArgumentException: Unknown level constant [WARNING].